packages/xstate-inspect/src/browser.ts

```diff
@@ -203,6 +203,7 @@ export function inspect(
       const observer = toObserver(next, onError, onComplete);
 
       listeners.add(observer);
+      observer.next(inspectService.state);
```

Comment:
an alternative fix would be to just use the existing `inspectService.subscribe` as this proxying doesn't give us a lot:
https://github.com/statelyai/xstate/blob/5010717d216fe74b36826b193f985b337b19f71a/packages/xstate-inspect/src/browser.ts#L92-L94

Reply:
I think your current fix makes the most sense